python/cudf/cudf/bindings/copying.pyx

```diff
@@ -163,3 +206,53 @@ def copy_column(input_col):
     free(output)
 
     return Column.from_mem_views(data, mask, output.null_count)
+
+
+def apply_copy_range(out_col, in_col, int out_begin, int out_end,
+                     int in_begin):
```

Comment:
You should be more explicit with types here as different platforms can have different sized integers, i.e. x86 vs PPC vs ARM.

Reply:
Will resolve when libcudf resolves as well.